sw/ground_segment/cockpit/live.ml

```diff
@@ -1521,6 +1532,45 @@ let get_shapes = fun (geomap:G.widget)_sender vs ->
 let listen_shapes = fun (geomap:G.widget) ->
   safe_bind "SHAPE" (get_shapes geomap)
 
+let get_cam_status = fun (geomap:MapCanvas.widget) _sender vs ->
```

Comment:
Even without a gimbal just a camera point is  not enough. An aircraft traveling north at a 30deg pitch, will point a camera is the same direction as an aircraft traveling east with a 30deg roll.